gubernator/config.yaml

```diff
@@ -34,7 +34,6 @@ jobs:
   - pull-kubernetes-e2e-kind
   - pull-kubernetes-e2e-kind-ipv6
   - pull-kubernetes-integration
-  - pull-kubernetes-kubemark-e2e-gce-big
```

Comment:
Given the job is not being removed, just no longer required, shouldn't we keep this as is?

Reply:
The recommended way to navigate to your job's logs would be to use spyglass.  Gubernator's list is generated based on `always_run` jobs.  We have basically stopped supporting gubernator at this point.